src/main/java/com/azure/cosmos/kafka/connect/source/CosmosDBSourceConfig.java

```diff
@@ -70,9 +70,9 @@
     private Long bufferSize;
     private Long batchSize;
     private Long pollInterval;
-    private String messageKeyEnabled;
+    private boolean messageKeyEnabled;
```

Comment:
i believed we got rid of boolean initially in our config b/c of recommendations in the Connect dev docs about not using booleans, but instead to use string based dropdown lists. 

is this no longer the case?

Reply:
@ryancrawcour  I'm not sure about the recommendations. We do use this pattern in JDBC connector as well (Boolean ConfigDef w/ String default value) . I just changed it so that I could remove the `EnumRecommender`. It seemed unnecessary to pull in schema registry client just for that. Can you point me to the docs that you are referencing and I can track down what our latest recommendation is?

Reply:
@ncliang fair enough. as long as the default works in the control center UX when using boolean, i am happy.